main/autocode/src/org/ejml/GenerateKotlinCode32.java

```diff
@@ -85,7 +85,7 @@ public GenerateKotlinCode32() {
         }
 
         converter.replacePattern("DScalar", "FScalar");
-        converter.replacePattern("ConvertD", "ConvertF");
```

Comment:
Seeing there is also a kotlin generator, I think I might have missed adding the `DOperator` prefix + converter there.
Or is this not necessary, e.g. how do these lists differ?

Reply:
Kotlin right now is just extensions. Not needed. Most users will probably never even need to type out `DOperator` since it will be implicit as a lambda.